frontend/components/mobile/MobileSearchBar.tsx

```
@@ -7,11 +7,11 @@ interface MobileSearchBarProps {
 export default function MobileSearchBar(props: MobileSearchBarProps) {
   return (
     <div
-      className="flex border border-1 border-core-outline rounded-full h-10 md:h-10 pl-5 pr-0.5 width w-fit items-center bg-white shadow-sm"
+      className="flex border border-1 border-core-outline rounded-full h-10 pl-5 pr-0.5 items-center bg-white shadow-sm w-full max-w-full min-w-0 cursor-pointer"
       onClick={props.openSearchInput}
     >
-      <h2 className="min-w-[120px] text-gray-500">Search events</h2>
-      <button className="w-7 h-7 rounded-full border border-black bg-black mr-1">
+      <h2 className="flex-1 text-gray-500 text-sm truncate">Search events</h2>
+      <button className="w-7 h-7 rounded-full border border-black bg-black mr-1 flex-shrink-0">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)